app/assets/javascripts/application.js

```diff
@@ -17,28 +15,16 @@
 //= require jquery.gridster
 //= require hidden_values
 //= require select_on_click
-//= require select2
```

Comment:
Maybe it's worth keeping this line then, or at least ensure select2 from npm runs at this time. Currently I'm getting `TypeError: $(...).select2 is not a function` when application.js calls `activate_select2(':root');`